images/linux/scripts/installers/google-chrome.sh

```diff
@@ -41,9 +36,4 @@ chmod +x $CHROMEDRIVER_BIN
 ln -s "$CHROMEDRIVER_BIN" /usr/bin/
 echo "CHROMEWEBDRIVER=$CHROMEDRIVER_DIR" | tee -a /etc/environment
 
-# Run tests to determine that the chromedriver installed as expected
-echo "Testing to make sure that script performed as expected, and basic scenarios work"
-if ! command -v chromedriver; then
-    echo "chromedriver was not installed"
-    exit 1
-fi
+invoke_tests "Browsers" "Firefox"
```

Comment:
I believe it should be "Chrome" here and "Firefox" in firefox.sh scripts, not a vice versa.

Reply:
Thank you!! Fixed now:)